model/_global/delete_pit/structures.smithy

```diff
@@ -7,20 +7,32 @@
 $version: "2"
 namespace OpenSearch
 
-@mixin
-structure DeletePit_QueryParams {
+list PitInfoList {
+    member: PitInfo
```

Comment:
We need to be careful of giving things too generic names as everything is within one namespace here. I think it'd make sense for this list and the below structure to be `DeletedPit` rather than `PitInfo`

Reply:
> We need to be careful of giving things too generic names as everything is within one namespace here. I think it'd make sense for this list and the below structure to be `DeletedPit` rather than `PitInfo`

Thank you, i got it. You can recheck now